src/main/plugins/org.dita.base/xsl/preprocess/maplinkImpl.xsl

```diff
@@ -119,17 +118,14 @@ See the accompanying LICENSE file for applicable license.
         <xsl:with-param name="pathFromMaplist" select="$pathFromMaplist"/>
       </xsl:call-template>
     </xsl:variable>
-    <!-- If going to print, and @print=no, do not create links for this topicref -->
-    <xsl:if test="not(($FINALOUTPUTTYPE = 'PDF' or $FINALOUTPUTTYPE = 'IDD') and @print = 'no')">
```

Comment:
The param is never set so
`not(($FINALOUTPUTTYPE = 'PDF' or $FINALOUTPUTTYPE = 'IDD') and @print = 'no')`
==>
`not((false() or false()) and @print = 'no')`
==>
`not((false()) and @print='no')`
==>
`not(false())`
==>
`<xsl if test="true()">`